crates/pink/pink-extension/src/lib.rs

```diff
@@ -19,10 +19,15 @@ pub mod predefined_accounts {
 
     // TODO.kevin: Should move to a separate crates. Maybe after https://github.com/Phala-Network/phala-blockchain/issues/861 resolved.
     pub const ACCOUNT_PALLET: [u8; 32] = *b"sys::pellet\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0";
+    pub const ACCOUNT_RUNTIME: [u8; 32] = [0; 32];
```

Comment:
Better to not use completely zero address. It's easy to accidentally pass a zero account, causing vulnerabilities

Reply:
Fixed